snitch.go

```diff
@@ -58,17 +58,26 @@ const (
 	// MaxWASMPayloadSize is the maximum size of data that can be sent to the WASM module
 	MaxWASMPayloadSize = 1024 * 1024 // 1Mi
 
-	// ClientTypeSDK & ClientTypeShim are referenced by shims and SDKs to specify
-	// which type of client they are upon registration with the snitch-server.
+	// ClientTypeSDK & ClientTypeShim are referenced by shims and SDKs to indicate
+	// in what context this SDK is being used.
 	ClientTypeSDK  ClientType = 1
 	ClientTypeShim ClientType = 2
+
+	// OperationTypeConsumer and OperationTypeProducer are used to indicate the
+	// type of operation the Process() call is performing.
+	OperationTypeConsumer OperationType = 1
```

Comment:
The available/valid ints were not defined before so user had to specify `1` or `2`. I think?

Reply:
They were defined before here, https://github.com/streamdal/snitch-go-client/pull/26/files#diff-6b5a0e6cb3e11066b02351f5616b23cb0b08387a28f69cb7ea9552822cacaaa6L47